doc/pset.mediawiki

```diff
@@ -255,6 +255,46 @@ The currently defined elements per-input proprietary types are as follows:
 |
 | 0
 | 2
+|-
+| Explicit Value
+| <tt>PSBT_ELEMENTS_IN_EXPLICIT_VALUE = 0x11</tt>
+| None
+| No key data
+| <tt><64-bit little endian int value></tt>
+| The explicit value for the input being spent. If provided, <tt>PSBT_ELEMENTS_IN_VALUE_PROOF</tt> must be provided too.
```

Comment:
In d1d7af3442059fd0b7a632ac0c23dbc2eafa94e3:

What if the input value is already explicit? Should we just forbid this field? I guess so.

Reply:
Same for the asset?

Reply:
Added a sentence saying it cannot be provided when the input already has an explicit value. Did the same for the asset field.